modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -1472,26 +1472,6 @@ public DefaultCodegen() {
 
         reservedWords = new HashSet<String>();
 
-        // TODO: Move Java specific import mappings out of DefaultCodegen.
-        importMapping = new HashMap<String, String>();
```

Comment:
I think many of these are used by Kotlin, and maybe also Groovy. I think the TODO should have read `JVM specific`. I think the only Java specific ones are the joda time.

Reply:
Updated for java, scala, kotlin, groovy